skglm/estimators.py

```diff
@@ -1159,6 +1162,169 @@ def fit(self, X, y):
     # TODO add predict_proba for LinearSVC
 
 
+class CoxEstimator(LinearModel):
+    r"""Elastic Cox estimator with Efron and Breslow estimate.
+
+    Refer to :ref:`Mathematics behind Cox datafit <maths_cox_datafit>`
+    for details about the datafit expression. The data convention for the estimator is
+
+    - ``X`` the design matrix with ``n_features`` predictors
+    - ``y`` a two-column vector where the first ``tm`` is of event time occurrences
+      and the second ``s`` is of censoring.
+
+    For L2 Cox, ``l1_ratio=0.``, :ref:`LBFGS <skglm.solvers.LBFGS>`
+    is the used solver, otherwise it is :ref:`ProxNewton <skglm.solvers.ProxNewton>`.
+
+    Parameters
+    ----------
+    alpha : float, optional
+        Penalty strength. It must positive ``alpha > 0``.
```

Comment:
```suggestion
        Penalty strength. It must strictly positive.
```

Reply:
As far as I know, _positive_ means being ``> 0``, whereas _non-negative_ means ``>= 0``.

cf. https://math.stackexchange.com/questions/18464/is-positive-the-same-as-non-negative

Reply:
yeah it's a shit way of naming things so let's go the french way